src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = (
+  { person: { name, age, sex, isMarried, partnerName } },
+) => {
+  const marriedWife = isMarried === true && sex === 'f';
+  const marriedHusband = isMarried === true && sex === 'm';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age > 0 && <p className="Person__age">{`I am ${age}`}</p>}
+      <p className="Person__partner">{marriedWife && `${partnerName} is my husband`}</p>
+      <p className="Person__partner">{marriedHusband && `${partnerName} is my wife`}</p>
+      <p className="Person__partner">{!isMarried && `I am not married`}</p>
```

Comment:
And then here you won't have to duplicate the p tags, and use a ternary instead:
```js
<p className="Person__partner">
  {isMarried
    ? `${partnerName} is my ${spouse}`
    : 'I am not married'}
</p>

Reply:
Good remark, please fix. You have empty p tag in your solution
![image](https://github.com/mate-academy/react_person/assets/54936044/93c8112a-8691-45a0-847a-479a772214f9)